cdc/sinkv2/metrics/statistics.go

```diff
@@ -125,6 +108,16 @@ func (b *Statistics) AddRowsCount(count int) {
 	atomic.AddUint64(&b.totalRows, uint64(count))
 }
 
+// SubRowsCount records total number of rows needs to flush
+func (b *Statistics) SubRowsCount(count int) {
```

Comment:
The function `SubRowsCount()` and `TotalRowsCount` are obsolete. Maybe we can delete them.